### PR TITLE
[react-dates]  Add types for phrases DateRangePickerPhrases

### PR DESCRIPTION
**Pull request by sinziananicolae**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://github.com/airbnb/react-dates/blob/4027557edb0e17b847cc936d93d9092b155beb85/src/defaultPhrases.js#L138

https://github.com/airbnb/react-dates/blob/4027557edb0e17b847cc936d93d9092b155beb85/src/utils/getPhrasePropTypes.js
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-dates/index.d.ts [hunk 1]
```diff
@@ -266,9 +266,34 @@ declare namespace ReactDates {
         // internationalization props
         monthFormat?: string,
         phrases?: {
-            focusStartDate: string,
-            clearDates: string,
-            keyboardNavigationInstructions: string,
+            calendarLabel?: string,
```
Comment (ArturAmpilogov) [inline types/react-dates/index.d.ts]: It seems to be a wrong Shape based on SingleDatePickerPhrases,
while here should be DayPickerPhrases  https://github.com/airbnb/react-dates/blob/master/src/defaultPhrases.js#L138,
can you please change?

**Review by ArturAmpilogov — CHANGES_REQUESTED** [review]

File: types/react-dates/index.d.ts (hunk 1 above)
Comment (sinziananicolae) [reply types/react-dates/index.d.ts]: @Artur-A you are right. I have made the change.

**Review by ArturAmpilogov — APPROVED** [review]
All is good. Thank you.

**Comment by DanielRosenwasser** [comment]
Sorry for the delay, and thanks for your patience!

**Pull request merged**